tilecloud_chain/tests/test_generate.py

```diff
@@ -749,7 +749,7 @@ def test_mapnik_grid_drop(self, l):
                 tiles=((5, 7), (7, 4)),
                 regex=True,
                 expected="""The tile generation of layer 'mapnik_grid_drop' is finish
-Nb generated tiles: 12
+Nb generated tiles: 2
```

Comment:
Always instantiating all tile stores to remove duplication means either that having a Mapnik grid layer in the config file (even if it is not used in a generation) will create the counter so early that other layers will count their metatile with it, or that the counter is instantiated too late to count Mapnik tiles that were generated and dropped. The latter seemed a more acceptable tradeoff